misago/permissions/hooks/check_locked_thread_permission.py

```diff
@@ -116,4 +116,4 @@ def __call__(
         return super().__call__(action, permissions, thread)
 
 
-check_post_in_closed_thread_permission_hook = CheckPostInClosedThreadPermissionHook()
+check_locked_thread_permission_hook = CheckPostInClosedThreadPermissionHook()
```

Comment:
Please change `CheckPostInClosedThreadPermission` to `CheckLockedThreadPermission` in this file.